scripts/shell.nix

```
@@ -0,0 +1,9 @@
+with (import <nixpkgs> { });
+mkShell {
+  name = "scripts-shell";
+  buildInputs = [
+    semver-tool
+    yq-go
+    chart-testing
```

The github action plugin installs 3 tools, might be worth pulling in all 3. [Look here](https://github.com/helm/chart-testing-action/blob/main/action.yml).

```suggestion
    chart-testing
    yamllint
    yamale
```

---

Well, it works without them. Check the latest commit's run.